.github/dependabot.yml

```diff
@@ -12,3 +12,9 @@ updates:
       interval: "weekly"
     commit-message:
       prefix: "chore"
+  - package-ecosystem: "docker"
+    directory: "/"
```

Comment:
This should only be enabled for charts in `manifest_staging`

Reply:
Just curious, I noticed you have a dockerfile in `"/"`:
https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/ed68f1895677a32095a58c6fecaa66b27a8c972a/Dockerfile

Any reason to not also have Dependabot watch that?